lib/fluent/plugin/elasticsearch_error_handler.rb

```diff
@@ -62,6 +62,10 @@ def handle_error(response)
       stats.each_pair { |key, value| msg << "#{value} #{key}" }
       @plugin.log.debug msg.join(', ')
     end
+    if stats[:successes] + stats[:duplicates] == bulk_message_count
+      @plugin.log.debug("retry succeeded - all #{bulk_message_count} records were successfully sent")
```

Comment:
Is this debug message necessary?  Don't we have the message above as well?

Reply:
The message will say something like `Indexed (op = create), 151 duplicates, 1 successes`
If that's ok, ok

Reply:
:+1:
I also like using info level.
And I also acceptable debug level because other messages are specified `debug` level.

Reply:
i beleve the pattern is something like `@plugin.log.on_info { @plugin.log.info "my message" }`